src/pages/settings/SettingForm.tsx

```diff
@@ -113,6 +115,8 @@ const SettingForm: FC<Props> = ({ option, value }) => {
             Save
           </Button>
         </Form>
+      ) : isRestricted ? (
+        <span>{value ? value : "-"}</span>
```

Comment:
Doesn't this work properly here?
```suggestion
        <span>{value ?? "-"}</span>
```